### ISSUE TITLE
api-method-should-specify-api-response could not detect when using `applyDecorators` function

### ISSUE THREAD

**Issue by han911976** [darraghoriordan/eslint-plugin-nestjs-typed#199]
Hi
I am trying to follow code here to reduce my copy-and-paste hassle (and improve tiny readability)
<https://docs.nestjs.com/openapi/operations#advanced-generic-apiresponse>

funcionally my code works, which allow me to group multi response decorators into single decorator

```js
export const ApiAuthUnauthorizedNotFoundResponse = () => {
  return applyDecorators(
    ApiUnauthorizedResponse({ type: UnauthorizedResponse }),
    ApiNotFoundResponse({ type: NotFoundResponse }),
  );
};
```

However, warning will occur 
![image](https://github.com/user-attachments/assets/1db65431-53a3-486e-9413-7a02384b9e3c)

I am also having swagger cli enabled, I think the `ApiOkRepsonse` is generated by the cli and do not require manual paste.

As a work around I can slient the error / add a `ApiOkResponse()` for each endpoint which makes me feel redundant -v-

I think this might be able to fix by implementing or extends the original `ApiResponse`?

I am new to nest, hopefully this is not a stupid question.

Thank you.

[api-method-should-specify-api-response](https://github.com/darraghoriordan/eslint-plugin-nestjs-typed/blob/main/src/docs/rules/api-method-should-specify-api-response.md)

**Comment by darraghoriordan** [comment]
Yea so I don't use the cli in my project, this rule is probably redundant if you're having an automated tool adding the thing that it is telling you you should add? so you can just turn it off in your eslint conifg

```
   rules: {
   "@darraghor/nestjs-typed/api-method-should-specify-api-response": "off",
   }
```
